crates/uv/src/main.rs

```diff
@@ -107,6 +107,11 @@ async fn run() -> Result<ExitStatus> {
         }
     };
 
+    // enable warning before workspace loading.
+    if !cli.global_args.quiet {
+        uv_warnings::enable();
+    }
+
```

Comment:
If the flag is not set here, warnings of workspace parsing failure will not be output.

Reply:
I noticed this bug when I was creating the test.

If it is better to separate them into separate PRs, I will remove the relevant process and some of the tests.

Reply:
Yeah, could you create a different PR for those please?